server/engine/src/test/java/org/eclipse/lsp/cobol/positive/FolderTextRegistry.java

```diff
@@ -80,11 +81,26 @@ private void collectFilesToTest(String folderPath) throws IOException {
     String updateListingFlag = getUpdateListingFlag();
     Files.walk(Paths.get(folderPath)).filter(Files::isRegularFile).forEach(this::processFile);
     processCopybooks(folderPath);
+    dialect = getDialect(folderPath);
     if (!Files.exists(Paths.get(getListingFileLocation(folderPath))) || !updateListingFlag.equals("false"))
       createListingSnap(snaps, folderPath);
     else snaps = readFromSnap(folderPath);
   }
 
+  private String getDialect(String folderPath) throws IOException {
+    return Files.walk(Paths.get(folderPath)).filter(Files::isRegularFile)
+            .filter(f -> f.getFileName().toString().equals("dialect"))
```

Comment:
It feels like it should be documented somehow

Reply:
Added documentation about the folder structure of Positive Test on java doc of [FolderTextRegistry.java](https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/pull/2044/files/26951d128785dc8965c1d7de3c179498e1335edc#diff-42c808fd4050226e636cc120ac5c01983ab10d2fa87452102d7a4447696f0803)